bof-request.md

```diff
@@ -0,0 +1,43 @@
+# Name: Media over QUIC
+## Description
+ 
+Media over QUIC (moq) will work on solution for a simple low-latency 
+media delivery protocol that is common across ingest and distribution. 
+This work may address use cases including live streaming, gaming, and 
+media conferencing and allows efficient scaling. The solution supports 
+both web browsers and simple devices that don’t contain embedded browsers. 
+
+
+This is merely a placeholder for the BOF request, the details of the charter are  
+currently being worked developed, so please consider this as a heads up for 
+Media Over QUIC BOF request.
+
+Fill in the details below. Keep items in the order they appear here.
+
+## Required Details
+- Status: (not) WG Forming
+- Responsible AD: TBD
+- BOF proponents: TBD
+- BOF chairs: TBD
+- Number of people expected to attend: 70
+- Length of session (1 or 2 hours): 2 hours
+- Conflicts (whole Areas and/or WGs)
+   - Chair Conflicts: TBD
+   - Technology Overlap: TBD
+   - Key Participant Conflict: TBD
+
+## Information for IAB/IESG
+To allow evaluation of your proposal, please include the following items:
+
```

Comment:
Here i would add a bit of history. Last IETF had non WG form , how many people at it. Since had had calls on charter with how many people, list major applications participating or sending drafts - do this in form of "There has been interest and participation in this work from developers associated with widely deployed applications including Twitter, Facebook, Webex .... "